Add a description to the HAVE_POPEN_PROTOTYPE AC_DEFINE so that this define is
included in config.h.

Upstream-Status: Submitted (via email)
Signed-off-by: Ross Burton <ross.burton@intel.com>


diff --git a/aclocal.m4 b/aclocal.m4
index 3f20573..ec5ca51 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -931,7 +931,7 @@ ac_cv_td_popen=no,
 ac_cv_td_popen=yes))
 AC_MSG_RESULT($ac_cv_td_popen)
 if test $ac_cv_td_popen = yes; then
-	AC_DEFINE(HAVE_POPEN_PROTOTYPE)
+	AC_DEFINE(HAVE_POPEN_PROTOTYPE,[1],[Conflicting popen prototype])
 fi
 ])dnl
 dnl ---------------------------------------------------------------------------
